src/radical/pilot/staging_directives.py

```diff
@@ -80,7 +90,8 @@ def expand_staging_directives(sds: Union[str, Dict[str, Any], List[str]]
 
             # sanity check on dict syntax
             valid_keys = ['source', 'target', 'action', 'flags', 'priority',
-                          'uid', 'prof_id']
+                          'uid', 'prof_id', 'exception', 'exception_detail']
```

Comment:
we do not use `exception` and `exception_detail` right now, maybe skip them then?

Reply:
Thanks, removed.